Setup/UpgradeData.php

```diff
@@ -66,6 +66,10 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
             $this->updateSchemaVersion244($setup);
         }
 
+        if (version_compare($context->getVersion(), '6.0.0', '<')) {
```

Comment:
How did you test this as you did not upgrade the setup version so this will never be triggered. What is your take on this are we adding all the scripts under 1 version upgrade this could break some development environments switching between branches and versions.

Reply:
I manually changed the setup version in my environment in order to test it, but did not commit it as we can do that in a release merge. Do you think we should change the setup version in this feature branch?

I do believe that all migration scripts related to this config UI feature should be included under the same version upgrade because some configs are interdependent. This one in particular will not break environments when changing to older versions or branches because it only adds a new config row when needed, that row won't be used on envs after a downgrade. I'll keep an eye to keep new changes backwards compatible where possible.

Reply:
Now using alpha versioning for this branch https://github.com/Adyen/adyen-magento2/pull/669/commits/5678b920e1ca79f6f5df2637e329123ac2cc452d